src/lib/core/CHIPConfig.h

```diff
@@ -1292,6 +1292,39 @@
 #define CHIP_CONFIG_NODE_ADDRESS_RESOLVE_TIMEOUT_MSECS (5000)
 #endif // CHIP_CONFIG_NODE_ADDRESS_RESOLVE_TIMEOUT_MSECS
 
+/**
+ *  @def CHIP_CONFIG_MCSP_RETRANS_TABLE_SIZE
+ *
+ *  @brief
+ *    Size of the retrans table for message counter synchronization protocol
+ *
+ */
+#ifndef CHIP_CONFIG_MCSP_RETRANS_TABLE_SIZE
+#define CHIP_CONFIG_MCSP_RETRANS_TABLE_SIZE 4
```

Comment:
Why reduce the retrans window size to 4?

Reply:
This will be removed